ext/LegendSpecFitsRecipesBaseExt.jl

```diff
@@ -1275,7 +1275,7 @@ end
 
 # recipe for the lq_drift_time_correction report
 
-@recipe function f(report::NamedTuple{(:lq_prehist, :lq_report, :drift_prehist, :drift_report, :lq_box, :drift_time_func, :DEP_left, :DEP_right)}, e_cal, dt_eff, lq_e_corr, plot_type::Symbol)
+@recipe function f(report::NamedTuple{(:lq_report, :drift_report, :lq_box, :drift_time_func, :DEP_left, :DEP_right)}, e_cal, dt_eff, lq_e_corr, plot_type::Symbol)
```

Comment:
What is the `plot_type` doing? Can this be a `kwarg`?

Reply:
`plot_type` specifies which one the different plots should be plottet. 
Should i implement this by picking one them as a default plot?